tests/integration/proxy/grpcproxy/cluster_test.go

```diff
@@ -90,6 +129,27 @@ func (cts *clusterproxyTestServer) close(t *testing.T) {
 	}
 }
 
+func registDelayDeleteMember(lg *zap.Logger, endpoints []string, addr string, delay time.Duration, ttl int, t *testing.T) chan struct{} {
+	cfg := clientv3.Config{
+		Endpoints:   endpoints,
+		DialTimeout: 5 * time.Second,
+	}
+	client, err := integration2.NewClient(t, cfg)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	grpcproxy.Register(lg, client, "test-prefix", addr, ttl)
+	time.Sleep(time.Second)
+	donec := make(chan struct{})
+	go func() {
+		time.Sleep(delay)
```

Comment:
what is the point this goroutine if it's just waiting? I think this can be removed

Reply:
this goroutine used to delay close the etcd client, if client is closed, lease will expire soon, then the key which lease attached will delete, it is complex, use DeleteEndpoint is more convenient